product_barcode_generator/wizard/product_barcode_mass_assignment_wiz_view.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2016 Factor Libre, soluciones tecnologicas
+     Copyright (C) 2016-TODAY Janire Olagibel <janire.olagibel@factorlibre.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+
+<openerp>
+    <data>
+
+        <record model="ir.ui.view" id="product_barcode_mass_assignment_wiz_form">
+            <field name="name">Assign Product Barcode</field>
```

Comment:
FYI, name key in unnecessary on view record, because it is auto generated.
(not a blocking point)